analysis/jira__daily_issue_status_category_analysis.sql

```diff
@@ -0,0 +1,63 @@
+with field_history as ( 
+  
+  select * 
+  from {{ ref('jira__daily_issue_field_history') }} 
+), 
+
+statuses as ( 
+
+  select * 
+  from {{ ref('stg_jira__status') }} 
+), 
+
+status_category as ( 
+
+  select * 
+  from {{ ref('stg_jira__status_category') }} 
+), 
+
+status_mapping as ( 
+
+  select 
+    statuses.status_id, 
+    statuses.status_name, 
+    status_category.status_category_id, 
+    case when lower(status_category.status_category_name) = 'to do' then 'To Do' 
+      when lower(status_category.status_category_name) = 'in progress' then 'In Progress' 
+      when lower(status_category.status_category_name) = 'done' then 'Done' 
+      else 'Other' 
+    end as status_category_name 
+  from statuses left join status_category 
```

Comment:
left join should be on its own line
```suggestion
  from statuses 
  left join status_category 
```